### PR TITLE
Add new apple podcasts tags to import

### PR DESCRIPTION
**Pull request by gcampo88**
#353

### CODE REVIEW

File: app/models/podcast_import.rb [hunk 1]
```diff
@@ -339,6 +343,7 @@ def create_episode(story, entry)
     create_attributes[:keywords] = (entry[:itunes_keywords] || '').split(',').map(&:strip)
     create_attributes[:position] = entry[:itunes_order]
     create_attributes[:url] = episode_url(entry) || distro.default_url(story)
+    create_attributes[:itunes_type] = entry[:itunes_episode_type] unless entry[:itunes_episode_type].blank?
```
Comment (cavis) [inline app/models/podcast_import.rb]: Why the blank check here, but not in the other fields?

File: test/models/podcast_import_test.rb [hunk 2]
```diff
@@ -226,4 +231,8 @@ def stub_requests
     with(body: /prxUri/,
          headers: { 'Authorization' => 'Bearer thisisnotatoken' }).
     to_return(status: 200, body: json_file('transistor_episode'), headers: {})
+
+stub_request(:get, "https://feeder.prx.org/api/v1/authorization/episodes/153e6ea8-6485-4d53-9c22-bd996d0b3b03").
```
Comment (cavis) [inline test/models/podcast_import_test.rb]: Why is this added now?

File: app/models/podcast_import.rb (hunk 1 above)
Comment (gcampo88) [reply app/models/podcast_import.rb]: no particular reason, really -- just that the other fields were inside a call to `create`

File: test/models/podcast_import_test.rb (hunk 2 above)
Comment (gcampo88) [reply test/models/podcast_import_test.rb]: none of the existing tests check the Feeder episode created for a story, so I had to stub that method to add this one on line 97: `f.distributions.first.get_episode.itunes_type.must_equal 'full'`

**Comment by cavis** [comment]
Makes good sense.  :+7: :participation_trophy_medal:

**Comment by gcampo88** [comment]
participation!
